Makefile

```diff
@@ -349,7 +350,7 @@ rm -rf $$TMP_DIR ;\
 endef
 
 .PHONY: bundle
-bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
```

Comment:
I think we need to use `$(OPERATOR_SDK)` here too. 

https://github.com/jaegertracing/jaeger-operator/runs/5291691840?check_suite_focus=true#step:6:142